rules/o365/credential_access_o365_potential_password_spraying_attack.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/12/01"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/12/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed O365 user authentication attempts from a single IP address, which could be indicative
```

Comment:
Should we include the meat behind the rule in the description? I wonder if it that's easier than leaving the interpretation to the user to go find in the rule.  I am not against the current description, just thinking out loud.

Example:
"Identifies a high number (25) of failed O365 user authentication attempts from a single IP address within 30 minutes"